libraries/textcomposer/impl/src/main/kotlin/io/element/android/libraries/textcomposer/components/markdown/MarkdownTextInput.kt

```diff
@@ -48,8 +54,33 @@ fun MarkdownTextInput(
     onTyping: (Boolean) -> Unit,
     onSuggestionReceived: (Suggestion?) -> Unit,
     richTextEditorStyle: RichTextEditorStyle,
+    onRichContentSelected: ((Uri) -> Unit)?,
 ) {
     val canUpdateState = !subcomposing
+
+    // Copied from io.element.android.wysiwyg.internal.utils.UriContentListener
+    class ReceiveUriContentListener(
```

Comment:
Maybe we should provide this to the RTE and not the other way around. I'll keep it in mind.